src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -106,6 +106,21 @@ type Todo @model @auth(rules: [{ allow: owner, ownerField: "author" }]) {
 }
 ```
 
+<Callout warning>
+
+  **By default, owners have full permissions on their records!**
```

Comment:
This sentence can be misunderstood as "Auth rules always apply the permissions applied via the operations parameter but owner auth works different with elevated privileges".

Suggestion
"By default, owners can reassign the `owner` of their existing record to another user."

Reply:
Agreed, updated to align w/ your recommendation here.